src/storage/auth.ts

```
@@ -0,0 +1,115 @@
+
+    tokenExpiration: (state: AuthState): number | null => {
+      if (!state.accessToken) return null;
+      try {
+        const decoded = jwtDecode<TokenClaims>(state.accessToken);
+        return decoded.exp * 1000;
+      } catch {
+        return null;
+      }
+    },
+
+    isTokenExpired: (state: AuthState): boolean => {
```

ESLint

[error] 'state' is defined but never used.

(@typescript-eslint/no-unused-vars)